NuimoSDK/Properties/AssemblyInfo.cs

```diff
@@ -24,5 +26,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: ComVisible(false)]
-[assembly: NeutralResourcesLanguage("")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
Is it necessary to specify version? It's not synchronized with the NuGet package version.